src/Native/build-native.sh

```diff
@@ -167,6 +167,46 @@ if [ $CPUName == "i686" ]; then
     __BuildArch=x86
 fi
 
+# Use uname to determine what the OS is.
+OSName=$(uname -s)
```

Comment:
Where where these getting set before? I generally like to avoid seeing these maps in our scripts which is why I'm asking these questions.